doc/developers-guide/Testing-MongooseIM.md

```diff
@@ -108,6 +108,30 @@ The command can be used instead:
 And `mam` is used to run `mam_SUITE` suite only.
 
 
+## Debugging big tests database
```

Comment:
absolutely love it.
it's needed probably twice a year but always time consuming to recall how to do it. thanks a lot